File: app/src/main/java/com/blog/demo/md/NavigationViewActivity.java
package com.blog.demo.md;

import android.app.Activity;
import android.os.Bundle;
import android.view.MenuItem;
import android.widget.Toast;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.drawerlayout.widget.DrawerLayout;

import com.blog.demo.R;
import com.google.android.material.navigation.NavigationView;

public class NavigationViewActivity extends Activity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_material_design_navigation_view);

        final DrawerLayout drawerLayout = findViewById(R.id.drawer_layout);
        NavigationView navigationView = findViewById(R.id.navigation_view);
        navigationView.setNavigationItemSelectedListener(new NavigationView.OnNavigationItemSelectedListener() {
            @Override
            public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
                if (menuItem.getItemId() == R.id.menu_address
                        || menuItem.getItemId() == R.id.menu_more) {
                    menuItem.setChecked(true);
                }
                Toast.makeText(NavigationViewActivity.this, menuItem.getTitle(), Toast.LENGTH_SHORT).show();
                menuItem.getTitle();

                drawerLayout.closeDrawers();
                return true;
            }
        });
    }

}

File: app/src/main/java/com/blog/demo/md/NavigationViewActivity.kt
package com.blog.demo.md

import android.app.Activity
import android.os.Bundle
import android.widget.Toast
import androidx.drawerlayout.widget.DrawerLayout
import com.blog.demo.R
import com.google.android.material.navigation.NavigationView

class NavigationViewActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_material_design_navigation_view)

        val drawerLayout: DrawerLayout = findViewById(R.id.drawer_layout)
        val navigationView: NavigationView = findViewById(R.id.navigation_view)

        navigationView.setNavigationItemSelectedListener { menuItem ->
            if (menuItem.itemId == R.id.menu_address || menuItem.itemId == R.id.menu_more) {
                menuItem.isChecked = true
            }
            Toast.makeText(this@NavigationViewActivity, menuItem.title, Toast.LENGTH_SHORT).show()
            menuItem.title
            drawerLayout.closeDrawers()
            true
        }
    }

}